boards/nrf52840dongle/include/board.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2019 Christian Amsüss <chrysn@fsfe.org>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_nrf52840dongle
+ * @{
+ *
+ * @file
+ * @brief       Board specific configuration for the nRF52840-Dongle
+ *
+ * @author      Christian Amsüss <chrysn@fsfe.org>
+ */
+
+#ifndef BOARD_H
+#define BOARD_H
+
+#include "cpu.h"
+#include "board_common.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    LED pin configuration
+ * @{
+ */
+ /** @brief The LED labelled LD1 */
+#define LED0_PIN            GPIO_PIN(0, 6)
+
```

Comment:
remove newline (also below LED3_PIN)

Reply:
They are deliberately grouped, as they are actually a single monochrome and three channels of an RGB LED. That might not be visible in the documentation, but I'd consider that a meaningful grouping.

Reply:
I see, makes sense.